receiver/README.md

```diff
@@ -151,22 +151,42 @@ receivers:
               - targets: ['localhost:9777']
 ```
 
-### Include Filter
-Include Filter provides ability to filter scraping metrics per target. If a filter is specified for
-a target then only those metrics which exactly matches one of the metrics specified in the `Include Filter` list will be scraped.
-Rest of the metrics from the targets will be dropped.
+### Job Export Prefix
+Job Export Prefix provides the ability to add a metrics prefix per target. If a prefix is specified for a target then
+those metrics are scraped from that target will have the prefix added in the exported name.
 
 #### Syntax
 - Endpoint should be double quoted.
-- Metrics should be specified in form of a list.
+- Prefix should be double quoted with a "/" at the end.
 
 #### Example
 ```yaml
 receivers:
     prometheus:
-      include_filter: {
-        "localhost:9777" : [http/server/server_latency, custom_metric1],
-        "localhost:9778" : [http/client/roundtrip_latency],                
+      job_export_prefix: {
+        "localhost:9777" : "opencensus.io/prometheus"
```

Comment:
If this is per target as well as the filters, why do we not extend Target to have more properties:
```
    prometheus:
      config:
        scrape_configs:
          - job_name: 'opencensus_service'
            scrape_interval: 5s
            static_configs:
              - target: 'localhost:8889'
              include_filter: [http/server/server_latency, custom_metric1]
              export_prefix: "opencensus.io/prometheus"
          - job_name: 'jdbc_apps'
            scrape_interval: 3s
            static_configs:
              - target: 'localhost:9777'
```

Reply:
The config portion of the specification comes from Prometheus - I don't think there is a simple way to extend it.

Reply:
Ok, let me first understand if the mapping I suggested makes sense. I mean if the prefix apply to the target inside a job not to the whole job.